netweaver/monitoring.sls

```diff
@@ -0,0 +1,30 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+
+prometheus_sap_host_exporter_pkg:
+  pkg.installed:
+    - name: prometheus-sap_host_exporter
+
+sap_host_exporter_exporter_service:
+  service.running:
+    - name: prometheus-sap_host_exporter
+    - enable: True
+    - restart: True
+    - require:
+      - pkg: prometheus_sap_host_exporter_pkg
+      - file: sap_host_exporter_configuration
+
+# for each node the instance number change so we need to adapt configuration
+{% for node in netweaver.nodes if host == node.host %}
+sap_host_exporter_configuration:
+  file.managed:
+    - name: /etc/sap_host_exporter.yaml
+    - contents: |
+         port: "9680"
+         addres: "0.0.0.0"
+         log-level: "info"
+         sap-control-url: "http://127.0.0.1:5{{ node.instance }}13"
+         sap-control-user: "{{ node.sid.lower() }}"
```

Comment:
You will need to change this `sap-control-user` line. The user is created as `"{{ node.sid.lower() }}adm"` (check that last `adm`)

Reply:
done thx